melodic/distribution.yaml

```diff
@@ -3766,6 +3766,31 @@ repositories:
       url: https://github.com/tu-darmstadt-ros-pkg/hector_slam.git
       version: melodic-devel
     status: maintained
+  heifu:
+    doc:
+      type: git
+      url: https://heifu:G2SzWMcydduaaXwttrxA@gitlab.pdmfc.com/drones/ros1/heifu.git
```

Comment:
Please update this to be a publicly accessible repository which does not require credentials to access. Not all tools can handle embedded tokens.